pipeline/runtime/common.go

```diff
@@ -155,7 +155,24 @@ func fetchOutputVariables(outputFile string, out io.Writer, isDotEnvFile bool) (
 		line := s.Text()
 		sa := strings.Split(line, delimiter)
 		if len(sa) < 2 { // nolint:gomnd
-			log.WithField("variable", sa[0]).Warnln("output variable does not exist")
+			// Fallback: try equals delimiter if space delimiter didn't work
+			if delimiter == outputDelimiterSpace {
+				if eqIndex := strings.Index(line, "="); eqIndex > 0 {
```

Comment:
what if the delimiter is space but the value contains '=', might produce some bad outcomes

Reply:
why is the delimiter coming as space if we want to use equals here?